device_services/generic/chargepoint.md

```diff
@@ -60,6 +65,22 @@ Chargepoint service is used to represent EV chargers.
 
 > Please note that `ready_to_charge` and `charging` are states required for charging control, while others have only informative value.
 
+* `phase_mode` defines allowed phase balancing modes, see table below for more details:
+
+| Value     | Input EVSE | Output EV  | Grid       |
+|-----------|------------|------------|------------|
+| `NL1L2L3` | N L1 L2 L3 | N L1 L2 L3 | `TN`, `TT` |
+| `NL1`     | N L1       | N L1       | `TN`, `TT` |
+| `NL2`     | N L2       | N L1       | `TN`, `TT` |
+| `NL3`     | N L3       | N L1       | `TN`, `TT` |
+| `L1L2L3`  | L1 L2 L3   | N L1 L2    | `IT`       |
+| `L1L2`    | L1 L2      | N L1       | `IT`       |
+| `L2L3`    | L2 L3      | N L1       | `IT`       |
+| `L3L1`    | L1 L3      | N L1       | `IT`       |
+
+> Based on device capabilities and its configuration a chargepoint may support only a subset of the above modes. 
+> For example a chargepoint in TN installation capable to switch between 3 phases and 1 phase only will support `NL1L2L3` and `NL1` modes.
```

Comment:
```suggestion
> For example a chargepoint in TN grid type capable to switch between either 3 phases or 1 phase will support `NL1L2L3` and `NL1` modes.
```

Reply:
See new version. Your rewording is not consistent with the original meaning.